src/addons/manager/addonmanager.cpp

```diff
@@ -174,11 +174,9 @@ bool AddonManager::loadManifest(const QString& manifestFileName) {
       }
       if (!enabled) {
         beginRemoveRows(QModelIndex(), pos, pos);
-        removeRow(pos);
         endRemoveRows();
       } else {
         beginInsertRows(QModelIndex(), pos, pos);
-        insertRow(pos);
```

Comment:
Why remove these two lines?

Reply:
But then what is the point of calling begin/end remove/insert rows if these functions are not being called anymore?